examples/IRMQTTServer/IRMQTTServer.h

```diff
@@ -46,6 +46,12 @@
 #define FILESYSTEM SPIFFS
 #endif  // defined(ESP8266)
 #endif  // FILESYSTEM
+
+#if (FILESYSTEM == LittleFS)
+#define FILESYSTEMSTR "LittleFS"
+#else
+#define FILESYSTEMSTR "SPIFFS"
+#endif
```

Comment:
Oops. My bad. I forgot it would require something like this. Still, it's better that way. Thanks.